xsd/siri_utility/siri_location-v2.0.xsd

```diff
@@ -141,31 +145,27 @@ Rail transport, Roads and road transport
    <xsd:documentation>GML Spatial coordinate reference system.</xsd:documentation>
   </xsd:annotation>
  </xsd:element>
- <!--====== LOCATIONe ============================================================-->
+ <!--====== LOCATION ============================================================-->
  <xsd:complexType name="LocationStructure">
   <xsd:annotation>
    <xsd:documentation>Type for gepspatial Position of a point. May be expressed in concrete WGS 84 Coordinates or any gml compatible point coordinates format.</xsd:documentation>
```

Comment:
"gepspatial" should be "geospatial"

Reply:
See Commit [abd9a81](https://github.com/SIRI-CEN/SIRI/pull/64/commits/abd9a81d05b94287158fe8d4e28029db2475e4de)